parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.java

```diff
@@ -57,17 +55,23 @@
  */
 public class ParselyTracker {
     private static ParselyTracker instance = null;
-    private static int DEFAULT_FLUSH_INTERVAL_SECS = 60;
-    private static int DEFAULT_ENGAGEMENT_INTERVAL_MILLIS = 10500;
+    private static final int DEFAULT_FLUSH_INTERVAL_SECS = 60;
+    private static final int DEFAULT_ENGAGEMENT_INTERVAL_MILLIS = 10500;
+    private static final int QUEUE_SIZE_LIMIT = 50;
+    private static final int STORAGE_SIZE_LIMIT = 100;
+    private static final String STORAGE_KEY = "parsely-events.ser";
+// emulator localhost
+//    private static final String ROOT_URL = "http://10.0.2.2:5001/";
+    private static final String ROOT_URL = "https://p1.parsely.com/";
     protected ArrayList<Map<String, Object>> eventQueue;
```

Comment:
Suggestion (💡): Consider splitting the static from the non-static fields with an empty line.

PS: This is an extra, feel free to ignore.